### PR TITLE
jan dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-january-2022.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+        - single binary
+        - eventual replacement for ~50 iCommands
+        - ls/put/get/rm/cp/repl/query
+        - https://github.com/irods/irods_client_cli
+
+    - [Logical Quotas](https://github.com/irods/irods_rule_engine_plugin_logical_quotas)
+
+    - [Hard Links Rule Engine Plugin](https://github.com/irods/irods_rule_engine_plugin_hard_links)
+        - awaiting more use cases before release
+        - https://github.com/irods/irods_rule_engine_plugin_hard_links
```
Comment (trel) [inline content/posts/irods-development-update-january-2022.md:249]: duplicate

File: content/posts/irods-development-update-january-2022.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+    - [AWS S3 Lambda](https://github.com/irods/irods_client_aws_lambda_s3)
+        - future release
+            - could brute-force confirm checksums to detect renames
+            - would persist metadata
+        - could use atomic database operations to increase batch size > 1
+        - https://github.com/irods/irods_client_aws_lambda_s3
```
Comment (trel) [inline content/posts/irods-development-update-january-2022.md:256]: duplicate

File: content/posts/irods-development-update-january-2022.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
+    - C++ S3 API
+        - working on packaging
+        - part of new Data Transfer Nodes pattern
+
+    - [iRODS Command Line Interface (CLI)](https://github.com/irods/irods_client_cli)
+        - single binary
+        - eventual replacement for ~50 iCommands
+        - ls/put/get/rm/cp/repl/query
+        - https://github.com/irods/irods_client_cli
```
Comment (trel) [inline content/posts/irods-development-update-january-2022.md:243]: duplicate

File: content/posts/irods-development-update-january-2022.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+    - SMBRODS project to surface iRODS as SMB
+
+    - CockroachDB Database Plugin
+
+    - Cloud Browser
+
+    - Member Ticketing System
+
+- Discussion
+
```
Comment (trel) [inline content/posts/irods-development-update-january-2022.md:274]: add
```
     - interest in HIPAA use cases / policy
         - CyVerse, Minnesota, NKI Amsterdam
         - best practices, dashboards, PEPs, common policies
         - sequestered, and then deletion policies
```

File: content/posts/irods-development-update-january-2022.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-january-2022.md]: done

File: content/posts/irods-development-update-january-2022.md (hunk 3 above)
Comment (korydraughn) [reply content/posts/irods-development-update-january-2022.md]: done

File: content/posts/irods-development-update-january-2022.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-january-2022.md]: done

File: content/posts/irods-development-update-january-2022.md (hunk 4 above)
Comment (korydraughn) [reply content/posts/irods-development-update-january-2022.md:271]: done

**Pull request merged**